src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -38,15 +36,25 @@
   @Parameter(property = "cloudSdkPath", required = false)
   protected File cloudSdkPath;
 
-  protected CloudSdk cloudSdk;
+  protected CloudSdk.Builder cloudSdkBuilder;
 
-  protected DefaultProcessRunner processRunner;
+  private final ProcessOutputLineListener gcloudOutputListener = new ProcessOutputLineListener() {
+    @Override
+    public void outputLine(String line) {
+      System.out.println("GCLOUD: " + line);
```

Comment:
the obvious choice is the Maven logger, but I still want to indicate that output is coming from GCLOUD somehow.